### ISSUE TITLE
Welcome email to new transcribers to include Steve & Eric's email addresses and invitation to join Transcribers Group (forum)

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/MyopicVicar#1510]

**Comment by edickens** [comment]
No.  Email addresses should not be put here.  The reason is not security, but the person answering may change and we do not want to have to then make a program change.
Why would they need to contact Steve and myslef?  It should be the Coordinator.  My only suggestion is that if they do not ger ta reply say within a week, then they should have a link that they can use to contact management via the Role in the website.
We do need some means to invite them to join the transcribers group.  At present it is up to me to manually do this in the GoogleGroup.  Hopefully in due course we will have our integrated forum.

**Comment by SteveBiggs** [comment]
Good point Eric. We need to give the new transcriber an easy way to contact someone else in the event that the coord does not get back to them. Perhaps the welcome email should state something like - "Your coordinator will be in touch, but if you do not hear from them within a week, please go to the website here xxxx where you will find details of who to contact."

**Comment by DeniseColbert** [comment]
Yes, that makes sense, thanks Eric.  We'd have to flesh out your roles a little on that page, to avoid people emailing Jo (unless we leave her off) but it would save a programming change if details are out of date.

**Comment by edickens** [comment]
Have just invited a new transcriber to join the transcribers group and got this, which I have not seen before:-
![Capture.JPG](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/542531935e93a002006d0247/125516c66e82c728ace21e0d46cb958a379791accda5fd53b652fa912a5265296b51f02e7384e671e6162351580d4bac43585f45ffac8928f0a42062c80d55b98e30063e937f03a6aa8318b500bf23884e8cbfb88b4bc4ef3d37aa.JPG)

**Comment by stsccfr** [comment]
Let's instead invite to join the transcriber group after the volunteer has completed training.
